javascript/code-challenges/multiBracketValidation/multi-bracket-validation.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+function multiBracketValidation(input) {
+  input.split('').forEach(element => {
+    if (!input.length) return false;
+    else if (element === '(' && element !== ')') return false;
+    else if (element === '{' && element !== '}') return false;
+    else if (element === '[' && element !== ']') return false;
+    else return true;
+  });
+}
+multiBracketValidation();
+
+
```

Comment:
This looks really great Tahmina, the only thing I will say if you could have the tests in a separate file that would be great!